JetUtilities/src/L2Creator.cc

```diff
@@ -1373,16 +1378,17 @@ void L2Creator::writeTextFileForCurrentAlgorithm_spline() {
 
                 //For expediency of Summer16_25nsV5_MC do eta-dependent clipping
                 fout<<setw(8) <<etamin<<setw(8)<<etamax
-                    <<setw(10)<<setprecision(6)<<(isection ? bounds.first : 0.001)
+                    <<setw(10)<<setprecision(6)<<(firstline ? 0.001 : bounds.first)
```

Comment:
Did you test this? Just want to make sure the logic is right.

Reply:
Yes, the new corrections are all done by this